### PR TITLE
GH action from 80sec to 20 sec

### PR DESCRIPTION
**Pull request by albertmink**
as long as we use from the super-linter suite only its
editorconfig feature, we can do editorconfig-checker directly

actually, super-linter action takes about 80 sec. Now it should be
faster

### CODE REVIEW

**Comment by larshp** [comment]
might need someone with access to settings to change the required checks?

image

**Comment by albertmink** [comment]
well, we might need an @SAP/abap-file-formats-admin to  adjust the GitHub actions setting -- see comment above

File: .github/workflows/editorconfig.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore: [main]
+    # Remove the line above to run when pushing to master
```
Comment (larshp) [inline .github/workflows/editorconfig.yml]:
```suggestion
```

**Comment by albertmink** [comment]
Add new action with this PR.
Delete the old one in a second PR.

**Review by larshp — APPROVED** [review]
looks good to me 👍

**Pull request merged**